Bottom: df8aaddf13923e5be9699ccce91f0c47bfc5f3a3
Top:    8d6bf8641096a3c39d78d518352add5d102babf1
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-05 15:02:46 -0700

fix-sbfile-add-a-bunch-of-tests


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 43930adc054..6c6d2eef6b6 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -158,7 +158,9 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             self.debugger.SetErrorFileHandle(f, False)
             self.handleCmd('lolwut', check=False, collect_result=False)
-            self.debugger.GetErrorFileHandle().write('FOOBAR\n')
+            f2 = self.debugger.GetErrorFileHandle()
+            f2.write('FOOBAR\n')
+            f2.flush()
         lldb.SBDebugger.Destroy(self.debugger)
         with open(self.out_filename, 'r') as f:
             errors = f.read()
@@ -182,12 +184,9 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             debugger.SetErrorFileHandle(f, False)
             self.handleCmd('lolwut', check=False, collect_result=False)
-            debugger.GetErrorFileHandle().write('FOOBAR\n')
         with open(self.out_filename, 'r') as f:
             errors = f.read()
             self.assertTrue(re.search(r'error:.*lolwut', errors))
-            self.assertTrue(re.search(r'FOOBAR', errors))
-
 
     @add_test_categories(['pyapi'])
     def test_sbfile_type_errors(self):
